docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -172,6 +172,14 @@ h|Supported values
 h|Kafka property name
 |`num.partitions`
 |===
+
+ifdef::community[]
```

Comment:
In this case, the 'topic authorization failed' message bears explanation, even without a link to the Red Hat service limits article. Therefore, I created both community and non-community versions.